docs/install.rst

```diff
@@ -15,15 +15,18 @@ Add `oauth2_provider` to your `INSTALLED_APPS`
     )
 
 
-If you need an OAuth2 provider you'll want to add the following to your urls.py
+Update your projects' `urls.py` to include this when you need an OAuth2 provider:
 
 .. code-block:: python
 
     urlpatterns = patterns(
         ...
-        url(r'^o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
+        url(r'^o/', include(('oauth2_provider.urls', 'oauth2_provider_app', ), namespace='oauth2_provider'), ),
```

Comment:
There's also this new way of configuring urlpatterns with `path`. Maybe we should mention it here?

Reply:
Also whitespace before closing bracket.